### ISSUE TITLE
Additional query parameter to specify duration

### ISSUE THREAD

**Issue by dschowta** [linksmart/historical-datastore#65]
In order to query for a relateive time, it is better to support `duration` like field in HDS query.
e.g. `/data/sensor1/reading?$duration=1h` will fetch the data of last 1 hour.

**Comment by farshidtz** [comment]
I think this is redundant. The client can simply provide start and end times to get the equivalent and more consistent results.

**Comment by dschowta** [comment]
If I remember it right, it was for the BIMERR project requirement, where the BIF data fetcher needed this functionality. The fetcher presumably could not calculate the absolute times.

**Comment by farshidtz** [comment]
Right, we wanted to set the URL in their UI for cron-based retrieval. They had no variable support for the URL.
